src/proxy/momento/src/protocol/memcache/get.rs

```diff
@@ -76,22 +76,22 @@ pub async fn get(
                     }
                 }
             }
-            Ok(Err(MomentoError::LimitExceeded(_))) => {
-                BACKEND_EX.increment();
-                BACKEND_EX_RATE_LIMITED.increment();
-            }
             Ok(Err(e)) => {
                 // we got some error from the momento client
                 // log and incr stats and move on treating it
                 // as a miss
-                error!("error for get: {}", e);
+                error!("backend error for get: {}", e);
```

Comment:
Do we need to differentiate backend error from proxy error? These are used interchangeably right now but I think they don't always agree.

Reply:
I think a user will want to know what component is having an issue. We're limited in what we can provide in the memcached protocol - we can only provide messages for `CLIENT_ERROR` and `SERVER_ERROR`. The plain `ERROR` response does not allow a message. I think here, the best we can do is try to ascribe the fault to a component in the log message and return a `SERVER_ERROR` with some description for both proxy and backend errors.

For RESP, I think we should try to put a descriptive message in the `-ERR` response.